### PR TITLE
Enable lgtm and close plugins for kubernetes org.

### PR DESCRIPTION
**Pull request by spxtr**
And added a check that we don't load duplicate plugins.

### CODE REVIEW

**Comment by k8s-oncall** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1248)

**Comment by spxtr** [comment]
```
--- FAIL: TestPlugins (0.00s)
	main_test.go:29: Could not load plugins: plugin close is duplicated for kubernetes/test-infra and kubernetes.
FAIL
exit status 1
FAIL	k8s.io/test-infra/prow/cmd/hook	0.052s
```

Tested by adding `close` back to `kubernetes/test-infra`.

File: prow/plugins/plugins.go [hunk 1]
```diff
@@ -86,13 +87,27 @@ func (pa *PluginAgent) Load(path string) error {
...
 				return fmt.Errorf("unknown plugin: %s", p)
 			}
 		}
 	}
+	// Check that there are no duplicates.
+	for k, v := range np {
+		if strings.Contains(k, "/") {
+			org := strings.Split(k, "/")[0]
+			for _, p1 := range v {
+				for _, p2 := range np[org] {
```
Comment (foxish) [inline prow/plugins/plugins.go]: Probably doesn't matter but maybe we should switch to using sets.String from k8s.io/pkg/util/sets.

**Comment by foxish** [comment]
LGTM except for the one comment.

**Comment by spxtr** [comment]
@foxish thanks for the comment. Mind if I submit as-is?

**Comment by foxish** [comment]
No problem at all. It's fine logically, just seemed a little hard to understand at first.

**Pull request merged**